Cargo.toml

```diff
@@ -59,7 +59,7 @@ opt-level = 3
 debug = 2
 debug-assertions = true  # TODO: test and disable if too slow
 overflow-checks = true
-lto = true
+lto = false
```

Comment:
why is this? can it be made optional just when fuzzing is enabled?

Reply:
This only affect the "fuzz" profile not normal builds. The change was needed because something has changed in nightly that breaks the build when building with gcov + lto.